thunar/thunar-location-bar.c

```diff
@@ -150,6 +158,26 @@ thunar_location_bar_init (ThunarLocationBar *bar)
 
 
 static void
+thunar_location_bar_finalize (GObject *object)
+{
+  ThunarLocationBar *bar = THUNAR_LOCATION_BAR (object);
+
+  _thunar_return_if_fail (THUNAR_IS_LOCATION_BAR (bar));
+
+  if( bar->locationEntry )
```

Comment:
Remember that in GTK Code Style (which Xfce loosely fools) the space comes before the parenthesis but not after. I won't be picky (yes, I'm annoying at work) this time, we can fix this later.